firewall/filesystem_storage_firewall.go

```diff
@@ -0,0 +1,41 @@
+package firewall
+
+import (
+	"errors"
+)
+
+type FilesystemAcraFirewall struct {
+
+	handlers []QueryHandler
+
+	storageDirectory string
+}
+
+
+var ErrHandlersNotSpecified = errors.New("at least one query handler should be specified")
+
+
+func NewFilesystemAcraFirewall(storageDirectory string) (*FilesystemAcraFirewall, error) {
+
+	return &FilesystemAcraFirewall{storageDirectory:storageDirectory}, nil
+}
+
+
+func (firewall *FilesystemAcraFirewall) AddSpecificHandler(handler QueryHandler){
```

Comment:
what do we mean by `specific`? if nothing then enough to name `AddHandler` I think or `AddQueryHandler`. It will be more informative

Reply:
Agree. Changed.